src/powershell-native/nativemsh/pwrshcommon/pwrshcommon.cpp

```diff
@@ -732,6 +732,7 @@ namespace NativeMsh
         "System.Globalization.Extensions",
         "System.IO",
         "System.IO.Compression",
```

Comment:
It seems `System.ComponentModel.Composition.dll` is gone. Then it should be removed from this list now.

Reply:
A note about `Microsoft.CodeAnalysis.VisualBasic.dll` we added from the `Add-Type` refactoring. Since we plan to remove the VB support, we shouldn't add that dll in this list.

Reply:
I'll take care of this